src/Card/README.md

```diff
@@ -51,34 +51,45 @@ This component uses a `Card` from react-bootstrap as a base component and extend
 )}
 ```
 
-## With muted styling
+## Card variants
 
-Use `muted` prop to show `Card` in inactive state.
+Use `variant` prop to use `Card` specific style variant.
 
 ```jsx live
 () => {
+  const [variant, setVariant] = useState('light');
   const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
 
   return (
-    <Card style={{ width: isExtraSmall ? "100%" : "18rem" }} muted>
-      <Card.ImageCap 
-        src="https://picsum.photos/360/200/"
-        srcAlt="Card image"
-      />
-      <Card.Header
-        title="Card Title"
+    <>
+      {/* start example form block */}
+      <ExamplePropsForm
```

Comment:
Note: it looks like the checkboxes in the `ExamplePropsForm` are functional (they switch between the variants), but they don't show when they are checked.

<img width="924" alt="image" src="https://user-images.githubusercontent.com/2828721/206716450-7f9fa6d6-327d-4662-bca4-e30f7ab149af.png">

Reply:
fixed, thanks! (it was because of another `ExamplePropsForm` we have on the page which conflicted with this one)